requirements.txt

```diff
@@ -6,4 +6,5 @@ numba
 sciris
 scirisweb
 gunicorn
-plotly_express
\ No newline at end of file
+plotly_express
+fire==0.3.1
```

Comment:
Btw, strong preference against pinning versions -- it's too easy to make libraries mutually incompatible that way. Suggest `fire>=0.3.1` and trust that Google won't break it ;)

Reply:
ok will keep this in mind for the next PR 👍